theme_graphene/views/customizations.xml

```diff
@@ -4,7 +4,7 @@
 <!-- ======== COVER ======== -->
 <template id="s_cover" inherit_id="website.s_cover" name="Graphene s_cover">
```

Comment:
Text is not readable

![image](https://github.com/user-attachments/assets/d65c22f5-8dce-4ab4-9b42-a7f1b488d937)